packages/ag-charts-community/src/chart/series/cartesian/scatterSeries.test.ts

```diff
@@ -97,4 +97,83 @@ describe('ScatterSeries', () => {
             });
         }
     });
+
+    describe('remove animation', () => {
```

Comment:
#223 just got merged, could you add/update this case and `bubbleSeries.test.ts` too so we don't end up with a gap between this PR and that one?